### PR TITLE
[Silicon Labs] Add Thunderboard Wear as a target board

### PR DESCRIPTION
**Pull request by stevew817**
Added Thunderboard Wear (based on EFM32GG) as a target for DAPLink.

### CODE REVIEW

File: projects.yaml [hunk 1]
```diff
@@ -129,6 +129,10 @@ projects:
         - *module_if
```
Comment (c1728p9) [inline projects.yaml]: The lpc11u35_efm32gg_stk_if should be removed since this is the thunderboard

Comment (stevew817) [inline projects.yaml]: Why? The Thunderboard is a different board from the GG STK.

Comment (c1728p9) [inline projects.yaml]: I thought you added this as a placeholder for the thunderboard

**Comment by c1728p9** [comment]
@mbed-bot: TEST

**Comment by stevew817** [comment]
Looks like something went wrong during CI testing... 'No test results found'?

**Comment by c1728p9** [comment]
Hi @stevew817, yeah this is a test issue that needs to be fixed on my end.

**Comment by c1728p9** [comment]
Hi @stevew817, this issue has been resolved.  Please rebase to master and I'll re-run testing

**Comment by stevew817** [comment]
@c1728p9 If you would be so kind to take care of the rebase, that'd be great.

**Comment by c1728p9** [comment]
I moved this to #123 and rebased it.  I also added the hooks for testing.

**Pull request closed without merging**